unified_targets/configs/SPRACINGH7NANO.config

```diff
@@ -0,0 +1,226 @@
+# Betaflight / SPRACINGH7NANO (SP7N) 4.1.0 Jul 10 2019 / 22:20:39 (8d37147db) MSP API: 1.42
+
+board_name SPRACINGH7NANO
+manufacturer_id SPRO
+
+# resources
+resource BEEPER 1 D07
+resource MOTOR 1 A00
+resource MOTOR 2 A01
+resource MOTOR 3 A02
+resource MOTOR 4 A03
+resource MOTOR 5 B06
+resource MOTOR 6 B07
+resource MOTOR 7 C06
+resource MOTOR 8 C07
+resource SERVO 1 NONE
```

Comment:
The lines containing `NONE` in here should be removed.

Reply:
done